.gitlab-ci.yml

```diff
@@ -1,113 +1,123 @@
+        ENVIRONMENT_NAME: development
     only:
         - master
 
-maven-build-production:
-    extends: .maven-build
+package-production:
+    extends: .package
     variables:
-        MAVEN_PROFILE: "production,$ACTIVE_EVAPRO,dgvapro"
-        ENVIRONMENT_NAME: "production"
+        MAVEN_PROFILE: production,$ACTIVE_EVAPRO
+        ENVIRONMENT_NAME: production
     only:
         - tags
 
+# Not executed, parent job definition for Tomcat deployments
+.deploy-tomcat:
+    stage: deploy
+    image: alpine:3.9.5
+    environment:
+        name: $ENVIRONMENT_NAME
+    script:
+        - DATETIME=$(date +%Y-%m-%dT%H-%M-%S)
+        - apk add --update curl
+        - curl -u $TOMCAT_USER:$TOMCAT_PASSWORD -T "$WS_ARTIFACT_PATH" "http://$TOMCAT_HOST/manager/text/deploy?update=true&path=/eva&version=$DATETIME" | grep "OK - Deployed application"
+
 deploy-tomcat-internal:
     extends: .deploy-tomcat
     variables:
-        TOMCAT_USER: "$TOMCAT_INTERNAL_USER"
-        TOMCAT_PASSWORD: "$TOMCAT_INTERNAL_PASSWORD"
-        TOMCAT_HOST: "$TOMCAT_INTERNAL_HOST"
-        ENVIRONMENT_NAME: "internal"
+        ENVIRONMENT_NAME: internal
+        TOMCAT_USER: $TOMCAT_INTERNAL_USER
+        TOMCAT_PASSWORD: $TOMCAT_INTERNAL_PASSWORD
+        TOMCAT_HOST: $TOMCAT_INTERNAL_HOST
+    dependencies:
+        - package-internal
     only:
         - master
 
 deploy-tomcat-development:
     extends: .deploy-tomcat
     variables:
-        TOMCAT_USER: "$TOMCAT_DEVELOPMENT_USER"
-        TOMCAT_PASSWORD: "$TOMCAT_DEVELOPMENT_PASSWORD"
-        TOMCAT_HOST: "$TOMCAT_DEVELOPMENT_HOST"
-        ENVIRONMENT_NAME: "development"
+        ENVIRONMENT_NAME: development
+        TOMCAT_USER: $TOMCAT_DEVELOPMENT_USER
+        TOMCAT_PASSWORD: $TOMCAT_DEVELOPMENT_PASSWORD
+        TOMCAT_HOST: $TOMCAT_DEVELOPMENT_HOST
+    dependencies:
+        - package-development
     only:
         - master
 
 deploy-tomcat-production:
     extends: .deploy-tomcat
     when: manual
     variables:
-        TOMCAT_USER: "$TOMCAT_PRODUCTION_USER"
-        TOMCAT_PASSWORD: "$TOMCAT_PRODUCTION_PASSWORD"
-        TOMCAT_HOST: "$TOMCAT_PRODUCTION_HOST"
-        ENVIRONMENT_NAME: "production"
+        ENVIRONMENT_NAME: production
+        TOMCAT_USER: $TOMCAT_PRODUCTION_USER
+        TOMCAT_PASSWORD: $TOMCAT_PRODUCTION_PASSWORD
+        TOMCAT_HOST: $TOMCAT_PRODUCTION_HOST
+    dependencies:
+        - package-production
     only:
         - tags
 
 deploy-tomcat-production-fallback:
     extends: .deploy-tomcat
     when: manual
     variables:
-        TOMCAT_USER: "$TOMCAT_PRODUCTION_FALLBACK_USER"
-        TOMCAT_PASSWORD: "$TOMCAT_PRODUCTION_FALLBACK_PASSWORD"
-        TOMCAT_HOST: "$TOMCAT_PRODUCTION_FALLBACK_HOST"
-        ENVIRONMENT_NAME: "production"
+        ENVIRONMENT_NAME: production
+        TOMCAT_USER: $TOMCAT_PRODUCTION_FALLBACK_USER
+        TOMCAT_PASSWORD: $TOMCAT_PRODUCTION_FALLBACK_PASSWORD
+        TOMCAT_HOST: $TOMCAT_PRODUCTION_FALLBACK_HOST
```

Comment:
Why aren't the double quotes needed any more?

Reply:
Quotes were used before but weren't really needed. I think it looks better without them so I removed them